sample_formatted_code.do_not_merge.c

```diff
@@ -0,0 +1,562 @@
+// Formatted using
+//   clang-format -i sample_formatted_code.do_not_merge.c
+
+// SortIncludes
+#include <stdio.h>
+#include <assert.h>
+#include <string.h>
+#include "sample.llvm.h"
+
+//--- Line Length ---
+// https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Coding_Style#Line-length
+// ColumnLimit: 80
```

Comment:
ColumnLimit is the config that impacts the formatting here. I typically mentioned most of them.

Reply:
I'm in favor of looking at relaxing the 80 chars. I think we've so far called this a soft limit and ~100 a hard limit. I'm okay with wider, where the main consideration should probably be the width of the review layout of github and gitlab.

Reply:
For what it's worth, Linux recently switched over to 100 chars.

Anyhow, clang-format only has a hard limit. I.e. whatever the number, that'll be the limit. Downside of making it bigger than the mostly used 80 chars is that it'll reformat to 100 chars some of the code that exceeded 80 chars.